documentation/backup.md

```diff
@@ -0,0 +1,49 @@
+# Backup Puppet Enterprise using the PEADM module
+
+## What is being backed up?
+
+By default the `peadm::backup` plan will backup the following items:
+
+1. Orchestrator
+2. PuppetDB
+3. RBAC
+   - Also, It copies the LDAP secret key if it present
+4. Activity
+5. Classification
+
+Optionally you can also backup:
+
+1. CA (CA and SSL certificates)
+
+**Note:** It is important to note that the `peadm::backup` plan in its current version is not as granular as you can get when you manually run [the `puppet-backup create` command.](https://puppet.com/docs/pe/2021.7/backing_up_and_restoring_pe.html#back_up_pe_infrastructure)
+
+## How can I customize my backup?
+
+We need to pass the `backup` parameter to the `peadm::backup` plan.
+
+**Example**
+
+Let's Backup _only_ RBAC
```

Comment:
we could offer an optional alternative way of calling this:
`bolt plan run peadm::backup  include="rbac"`
`bolt plan run peadm::backup  include="rbac,classifier"`

Reply:
Marking this as **feature-request**